src/hooks/useLocalCurrencyPrice.ts

```diff
@@ -0,0 +1,27 @@
+import { DEFAULT_LOCAL_CURRENCY } from 'constants/localCurrencies'
+import { useLocalCurrencyConversionRate } from 'graphql/data/ConversionRate'
+
+import { useActiveLocalCurrency } from './useActiveLocalCurrency'
+import { useUSDPrice } from './useUSDPrice'
+
+type useUSDPriceParameters = Parameters<typeof useUSDPrice>
+
+export function useLocalCurrencyPrice(...useUSDPriceParameters: useUSDPriceParameters) {
+  const activeLocalCurrency = useActiveLocalCurrency()
+  const activeLocalCurrencyIsUSD = activeLocalCurrency === DEFAULT_LOCAL_CURRENCY
+
+  const { data: usdPrice, isLoading: isUSDPriceLoading } = useUSDPrice(...useUSDPriceParameters)
+  const { data: localCurrencyConversionRate, isLoading: isLocalCurrencyConversionRateLoading } =
+    useLocalCurrencyConversionRate(activeLocalCurrency, activeLocalCurrencyIsUSD)
+
+  if (activeLocalCurrencyIsUSD) {
+    return { data: usdPrice, isLoading: isUSDPriceLoading }
+  }
```

Comment:
if we are currently in usd no need to fetch conversion rates so we just return what we got from the `useUSDPrice` hook